### ISSUE TITLE
Mstore pro

### ISSUE THREAD

**Issue by marknestor261** [inspireui/support#4356]
**_Step 1 (require): describe detail issues & screenshots_**
+ Detail Issues: 1. The left menu doesn't toggle when tapped, when running on Android.
      2. I correctly placed the launch screen images in their right sizes and folders but the splash screen appears cut.
    3. How do I integrate sms login because Facebook account kit mentioned in the documentation doesn't exist.
  4. Also how do I add flash sale section, tried it before using "layout bannersale" but could not display anything.
+ Product & version: latest(4.0.0)
+ Flutter (or React Native) version:  react-native-cli: 2.0.1.  react-native: 0.62.2
+ Testing Device/Simulator: Tecno Camon 11 Air (Android)
+ Screenshot issues :
![Screenshot_20200725-023259](https://user-images.githubusercontent.com/46901102/88447832-67079380-ce40-11ea-9300-21898f95abde.png)
![Screenshot_20200725-062021](https://user-images.githubusercontent.com/46901102/88447837-6838c080-ce40-11ea-91c0-e0a12180593d.png)

**_Step 2 (require): submit proof of purchasing the license on http://verify.inspireui.com_**

**_Important Note:_**
- Please help to ready the support policies before creating a new ticket: https://inspireui.com/support-policies/
- Kindly create only *One Ticket* & includes all the issues, that would help us focus to resolve it better. 
- If your all ticket was resolved & closed, but want to create a new ticket, just do simple step by linking to the previous verified ID (you don't need to submit the form again), for example, #Ticket-ID

Thank you so much for your time 😊

**Comment by codigojp2020** [comment]
Hello, I have the same problems, could someone help please?
nor does it show content in versions 9 or higher

**Comment by luyx2412** [comment]
Hi @marknestor261 
- Can you send me source code, i will check about the side menu. Because in my side everything workede normal
![image](https://user-images.githubusercontent.com/8909619/88471114-26d30e80-cf2f-11ea-9a80-4f79fd4ec616.png)
- Please follow the article which guide how to add splash screen https://medium.com/@appstud/add-a-splash-screen-to-a-react-native-app-810492e773f9
-  You can use firebase sms https://rnfirebase.io/auth/phone-auth
- Please replace your url in AppConfig.json to show flash sale. 
![image](https://user-images.githubusercontent.com/8909619/88471137-76193f00-cf2f-11ea-8c0f-66d34ba42e4c.png)

Thanks

**Comment by marknestor261** [comment]
1. The menu issue is resolved
2. I will follow firebase doc to setup, thanks
3.i will work of the flash sale too,
 One more thing:
On tab menu, the my orders icon is missing. I have attached a screenshot.
![Screenshot_20200728-125045](https://user-images.githubusercontent.com/46901102/88650047-0c389b00-d0d1-11ea-8bd5-34917d2756ae.png)

**Comment by luyx2412** [comment]
Hi @marknestor261 Please see in `src/navigation/index.js` line 260. You will see the my order icon, so please follow same it to import your order icon.
![image](https://user-images.githubusercontent.com/8909619/88674450-07490b00-d114-11ea-99ce-7eac84bb8a0f.png)

Thanks

**Comment by marknestor261** [comment]
Hello, i have checked but no clue of placing the myorder icon.
2. Also, i setup facebook login but when user clicks "continue as ..." it takes them no where but back to the login page.
3. How do i create countdown for flashsale.

**Comment by minhcasi** [comment]
@luyx2412 - please help to check on this issue ;)

**Comment by luyx2412** [comment]
Hi @marknestor261 Please send your source code via inspireui.team@gmail.com. Let me investigate about the issues.
2. Currently we did not support this feature, you can use the library component https://www.npmjs.com/package/react-native-countdown-component

Thanks

**Comment by marknestor261** [comment]
Hello, I have sent the source code with this email:
marknestor261@gmail.com

**Comment by luyx2412** [comment]
Hi @marknestor261 I just tested your app and everything worked normal. And the sms login you need test on real device and please make sure you add right SHA1 for both debug and release mode.
![image](https://user-images.githubusercontent.com/8909619/89426883-d052a500-d764-11ea-8dca-e4788ff4f35f.png)

Thanks

**Comment by marknestor261** [comment]
Hello, i am going to set sha1 in firebase. 
What about the settings page, how do i fix the app crashing when i tap certain selections.

**Comment by marknestor261** [comment]
Hello, I added SHA1 for both release and debug and downloaded new Google Jason file and built but still cannot login with Facebook and sms. Sms says verify fail and Facebook just goes back to login screen after I click continue as...
![Screenshot_20200806-110933](https://user-images.githubusercontent.com/46901102/89511999-485bb200-d7db-11ea-914f-0f7e4ef3da2f.png)

**Comment by luyx2412** [comment]
Hi @marknestor261 Please send your latest project via inspireui.team@gmail.com with title `4356`. Let me investigate and resend you.

Thanks

**Comment by marknestor261** [comment]
Hi @luyx2412 , I already sent my source code to the above email using my email marknestor261@gmail.com. I did not make any code changes, only Google Json file which I have also sent to the above email to replace. Thanks

**Comment by luyx2412** [comment]
Hi @marknestor261 I just checked your project and your config. Everything worked normal. So can you check again. https://drive.google.com/file/d/1ifkap3xVKXYS6qenypzWdsPSkUKSbpow/view?usp=sharing

Thanks

**Comment by marknestor261** [comment]
Hi @luyx2412 , now I don't understand why this side it gives errors. Let me try again and see.
What about Facebook login, did it work?.
And also the settings page, the crashing issue.

**Comment by luyx2412** [comment]
HI @marknestor261 Everything worked normal. Can u share the screenshot issue or video which investigate easier.

Thanks

**Comment by marknestor261** [comment]
Hi @luyx2412 I have sent the video to your email.

**Comment by luyx2412** [comment]
Hi @marknestor261 Please check again your side. I just tested about us screen which worked normal. Because your website load slow, so please wait it.
![image](https://user-images.githubusercontent.com/8909619/90408343-e734b800-e0d1-11ea-94ae-7e0529e891e3.png)

Thanks
